Registry/RegistryParse.py

```diff
@@ -322,15 +380,26 @@ def checksum(self):
 
     def validate_checksum(self):
         """
-        Is the file checksum valid.
+        Is the file checksum valid?
         """
         return self.calculate_checksum() == self.checksum()
 
-    def validate(self):
```

Comment:
I agree, for the same reason as above. Lets log a warning, and plan to remove this function in the 2.x release.